builder/amazon/common/step_deregister_ami.go

```diff
@@ -19,12 +20,25 @@ func (s *StepDeregisterAMI) Run(state multistep.StateBag) multistep.StepAction {
 	ui := state.Get("ui").(packer.Ui)
 
 	// check for force deregister
+	// owner-alias
 	if s.ForceDeregister {
```

Comment:
Here let's add a check for `if len(s.ForceDeregisterOwners) > 0` so if you have set either option we will kick off this logic. That way you don't need to set both options and the UX is less complicated.